### PR TITLE
Add version of SDL2 package with MSVC release build fix.

### PR DESCRIPTION
**Pull request by Predelnik**

### CODE REVIEW

File: cmake/projects/SDL2/hunter.cmake [hunk 1]
```diff
@@ -6,6 +6,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 include(hunter_cacheable)
 
+hunter_add_version(
+        PACKAGE_NAME
+        SDL2
+        VERSION
+        "2.0.4-p6"
```
Comment (ruslo) [inline cmake/projects/SDL2/hunter.cmake]: Add this version to `cmake/configs/default.cmake`:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#set-default-version

Comment (Predelnik) [reply cmake/projects/SDL2/hunter.cmake]: Done

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2907
* https://travis-ci.org/ingenue/hunter/builds/310944623?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2908
* https://travis-ci.org/ingenue/hunter/builds/310956330?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.177

**Pull request merged**